Atomic/objects/image.py

```diff
@@ -317,11 +317,17 @@ def is_system_type(self):
         return False
 
 def convert_size(size):
+    size_name = ("B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB")
     if size > 0:
-        size_name = ("B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB")
-        i = int(math.floor(math.log(size, 1000)))
-        p = math.pow(1000, i)
-        s = round(size/p, 2) # pylint: disable=round-builtin,old-division
-        if s > 0:
-            return '%s %s' % (s, size_name[i])
+        try:
+            i = int(math.floor(math.log(size, 1000)))
```

Comment:
Should these 1000's be 1024?  If I put in a size of 1024 to the function, I get 1.02KB back when I thought I'd just get 1.  For 1024*1024 instead of 1MB it returns 1.05MB.

Reply:
It's been that way? I could change if people prefer...